.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Checklist
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**PR Buddy:** _@ mention a team member_
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide historical context and customer-specific info._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any AMD hardware gotchas._
+
```

Comment:
AMD hardware gotchas cannot be put in PR. We can put a link to JIRA

Reply:
I don't think hardware gotchas are necessarily proprietary info?

Reply:
If AMD has published an errata, then yes, we can put a link to the errata. Otherwise, I don't think we should mention anything in public PR

Reply:
If that hardware "particularities" is already in public documentation, then yes.